apps/spruce/src/pages/distroSettings/tabs/GeneralTab/getFormSchema.ts

```diff
@@ -4,6 +4,7 @@ import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
 export const getFormSchema = (
   isContainerDistro: boolean,
   minimumHosts: number,
+  warningCopy: string,
```

Comment:
nit.
```suggestion
  singleTaskDistroWarnings: string[],
```
and just passing in the array instead of constructing it later.